avaframe/com1DFA/damCom1DFA.pyx

```diff
@@ -438,8 +438,8 @@ def initializeWallLines(cfg, dem, wallLineDict, savePath=''):
   cdef double w[4]
   cdef int Lx0, Ly0
   if wallLineDict is not None:
-    log.info('Initializing dam line from: %s' % str(wallLineDict['fileName'][0]))
-    log.info('Dam line feature: %s' % str(wallLineDict['Name'][0]))
+    log.debug('Initializing dam line from: %s' % str(wallLineDict['fileName'][0]))
+    log.debug('Dam line feature: %s' % str(wallLineDict['Name'][0]))
     # get z coordinate of the dam polyline
```

Comment:
Is there still some info that a dam is initialized and used in the log if all these are set to debug? I think one info like for the secondary release areas would be good

Reply:
mh yes, but it needs to be done outside of the pyx file. I'll start an issue...